docs/smart-contracts/01-intro.md

```diff
@@ -25,7 +25,7 @@ Source code for all Metropolis contracts can be found in this [GitHub repository
 | ControllerV1.4            | 0x4C98aF741e352C6551BfF9509b3f8ca9Dd4E6397  |
 
 ### Testnet Deployments
-##### Rinkeby - 4
+##### Goerli - 5
```

Comment:
@jtlacci do we have the new contract addresses for goerli? I'll need to add those in here